.travis.yml

```diff
@@ -16,10 +16,10 @@ script: "./.travis.sh"
 jobs:
   include:
     - stage: GitHub Release
-      if: tag IS present
-      d: ldc
+      #if: tag IS present
+      d: ldc-1.8.0
```

Comment:
Locking the release compiler. Just in case there are any regressions coming up (I experienced some with the dlang-tour and it was a hell to figure out that the sudden failures were due to an automatic compiler upgrade)